app/test/app.test.js

```diff
@@ -64,11 +64,40 @@ test('when post id is string', async () => {
 test('when id exists', async () => {
     let newPost = new postModel.PostEntity(null, 'title', 'slug', 'content');
     await postModel.insertPost(newPost);
-    
+
     let result = request('GET', `http://localhost:8080/posts/${newPost.id}`);
     expect(await result.statusCode).toBe(200);
     let payload = JSON.parse(result.body);
     expect(await payload.id).toEqual(newPost.id);
 
+    let resultUpdate = request('PUT', `http://localhost:8080/posts/${newPost.id}`, {
+        json: {
+            title: '1',
+            slug: '2',
+            content: '3'
+        }
+    });
+    expect(await result.statusCode).toBe(200);
+    let payload1 = JSON.parse(resultUpdate.body);
+    expect(await payload1.slug).toEqual('2');
+
     await postModel.deletePost(newPost);
 })
+
+test('updatePost when slug is not a string type', async () => {
+    let newPost = new postModel.PostEntity(null, 'title', 'slug', 'content');
+    await postModel.insertPost(newPost);
+
+    let result = request('PUT', `http://localhost:8080/posts/${newPost.id}`, {
+        json: {
+            title: '1',
+            slug: 2,
+            content: '3'
+        }
+    });
+    expect(await result.statusCode).toBe(400);
+    let payload = JSON.parse(result.body);
+    expect(await payload.errors).toEqual(['slug is not a string type']);
```

Comment:
Jeśli zrobisz konkretne testy walidatora to tutaj już nie będziesz musiała robić takich asercji